tensorflow/lite/build_def.bzl

```diff
@@ -3,6 +3,6 @@ def tflite_copts():
     copts = [
         "-DFARMHASH_NO_CXX_STRING",
         "-Wno-sign-compare",
-        "-fno-exceptions",  # Exceptions are unused in TFLite.
+        "-fno-exceptions",   #Exceptions are unused in TFLite.
```

Comment:
Is this consistent with other cases where we use comments in build files?

Reply:
+1 to Ting

Reply:
undoing change didn't check other  build files before changing format.